### PR TITLE
improve switch validation

### PR DESCRIPTION
**Pull request by rogerwim**

### CODE REVIEW

**Comment by rogerwim** [comment]
single line of code PR

**Review by rogerwim — COMMENTED** [review]
what happens if i write here? is this botted

**Review by watchful5406 — APPROVED** [review]
this is fine

File: server/client_events/client_switch_parameters_update_event.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def handle(data):
 
     if type(switches_updated) == dict and len(switches_updated) >= 1:
         for switch in switches_updated:
-            if switch in model.switches:
+            if model.switches.get(switch) is not None and model.switches.get(switch).get("position") is not None:
```
Comment (rogerwim) [inline server/client_events/client_switch_parameters_update_event.py:16]: single code change, makes this more resilant

**Pull request merged**